docs/contributor/arch/022-dynatrace-cummulative-to-delta-support.md

```diff
@@ -99,14 +99,19 @@ spec:
   - Users may not expect transformation logic to appear here. 
   - Similar to `Option 1`, it introduces complexity and documentation requirements. 
   - May not align well with the separation of concerns in the telemetry pipeline design.
-  - The `output.otlp` API designed to share across all signal types, which may lead to confusion if users expect it to apply only to metrics.
+  - The `output.otlp` API is designed to share across all signal types `logs`, `traces`, and `metrics`, which may lead to confusion if users expect it to apply only to metrics.
```

Comment:
The API is designed to share *what* across all signal types? T
his sentence seems incomplete.

Reply:
Oh! Well if it's passive, then rephrase the verb instead of adding what it shares.

> The `output.otlp` API is designed to be shared across...